systemtests/src/main/java/com/github/streamshub/systemtests/utils/WaitUtils.java

```diff
@@ -89,6 +90,23 @@ public static void waitForPodsReady(String namespaceName, LabelSelector selector
             }, onTimeout);
     }
 
+
+    public static void waitForPodRoll(String namespace, String podName, Map<String, String> podSnapshot) {
+        waitForPodRoll(namespace, podName, podSnapshot, TestFrameConstants.GLOBAL_POLL_INTERVAL_MEDIUM, TestFrameConstants.GLOBAL_TIMEOUT);
+    }
+
+    public static void waitForPodRoll(String namespace, String podName, Map<String, String> podSnapshots, long poll, long timeout) {
+        Wait.until("Pod to roll " + namespace + "/" + podName, poll, timeout,
+            () -> {
+                Pod pod = ResourceUtils.getKubeResource(Pod.class, namespace, podName);
+                if (pod != null) {
+                    return !Objects.equals(podSnapshots.get(podName), pod.getMetadata().getUid());
+                }
+                return false;
+            }
+        );
+    }
```

Comment:
So are these used anywhere?

Reply:
Now that i look at it, they are not, thanks!